.gitignore

```diff
@@ -1,14 +1,17 @@
+# Temporary cache
+.cache/
+
+# Node.js modules
 node_modules/
-npm-debug.log
-.DS_Store
-preview/
-tmp/
```

Comment:
**suggestion** It's a nice thing to know that you can use the `tmp` folder at the root of the project without worry. I'd be keen to keep it.

Reply:
Yeah let's keep it if anyone's using it otherwise let's get rid

If you'd rather keep it shall we use `.tmp` so it stays obvious at the top (A-Z) along with `.cache`?

Reply:
Good idea, let's go with the leading dot 😄